server.Dockerfile

```diff
@@ -30,7 +30,6 @@ FROM ${BASE_SERVER_IMAGE} as temporal-server
 WORKDIR /etc/temporal
 
 ENV TEMPORAL_HOME /etc/temporal
-ENV SERVICES "history:matching:frontend:worker"
```

Comment:
`SERVICES` env still can be passed with `docker run --env` even if it is not listed here, right?

Also I noticed that `ENV MY_VAR my-value` is deprecated format. Can you change other ENVs to `ENV <key>=<value>` format too?

Reply:
Yes, any env vars can be added when running the container